tests/pytorch/mock_sparse/test_mm.py

```diff
@@ -0,0 +1,267 @@
+import torch
+
+import backend as F
+
+from dgl.mock_sparse import create_from_coo, diag, bspmm
+
+def test_sparse_dense_mm():
+    dev = F.ctx()
+    # A: shape (N, M), X: shape (M, F)
+    row = torch.tensor([0, 1, 1]).to(dev)
+    col = torch.tensor([1, 0, 1]).to(dev)
+    val = torch.randn(len(row)).to(dev)
+    A = create_from_coo(row, col, val)
+    X = torch.randn(2, 3).to(dev)
+    sparse_result = A @ X
+    dense_result = A.adj.to_dense() @ X
```

Comment:
I see these tests are using the internal members of `A`. The issue is that when we replace the implementation in the future, we need to change test codes too. Can you directly use `row` and `col` to construct ground truth result?